recipes/boltzmannclean/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "boltzmannclean" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "e4b4ed0a7ae9c1f3dfb9283c87fd1c846d801934f16afcf0e09ce5f24bc8d073" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pandas
+    - numpy
+    - scipy
+    - scikit-learn
+
+test:
+  imports:
+    - boltzmannclean
+
+about:
+  home: https://faculty.ai
```

Comment:
I would prefer if the `home` address is `https://github.com/facultyai/boltzmannclean` - the current one is too general.